### PR TITLE
Fast Fire again

### LINKED ISSUE #1: improving FastFire check
Hey Jigglypuff here,

I'm not quite satisfied with the CheckFastFire(). What I hoped to test for is, if some player shoots every Server()->Tick() (should be 1ms). However I only got a minimum diff of 6ms between firings. This is too much, since it can also be manually achieved when trying hard. As a compromise I collect 30 samples and then test how much the timediffs differ from each other (not that correct :/ I know).

I'm also having a ping of 20ms when playing on localhost, so something is twisted with my machine. On occasion maybe you could test what's the smallest timediff for you...

### LINKED ISSUE #7: Improve regular fire detection
Do not ban players just for rapid-fire if they do not use other cheats.

### PR DESCRIPTION
**Pull request by derselbst**
* Fixed fast fire detection, only print it on rcon for now, so it should be safe to deploy these changes. Fixes #7 Fixes #1
* extend status command infos
* Contains a disclosed file, see my mail

When I tested it, for some reason `CheckInputPos()` didnt work for me and I got no warnings at all... but I couldnt find any relevant source code changes. So if you cant confirm this, I probably screwed up my setup. In any case pls let me know.

### CODE REVIEW

**Comment by derselbst** [comment]
\*and ofc I meant undisclosed

**Comment by black-roland** [comment]
I'll try to merge this all and deploy asap. Deploy relies on GitHub repo and it is hard to deploy testing build.

**Comment by derselbst** [comment]
> When I tested it, for some reason CheckInputPos() didnt work for me

Ok, just saw some "aimed exactly at" msgs floating around. So I must have messed up my setup :)

**Comment by black-roland** [comment]
Deployed with two latest commits ;)

File: src/engine/server/server.cpp [hunk 1]
```diff
@@ -1463,8 +1463,8 @@ void CServer::ConStatus(IConsole::IResult *pResult, void *pUser)
 
 				CGameContext* gserver = dynamic_cast<CGameContext*>(pThis->m_pGameServer);
 
-				str_format(aBuf, sizeof(aBuf), "id=%d addr=%s name='%s' score=%d warns=%d %s", i, aAddrStr,
-					pThis->m_aClients[i].m_aName, pThis->m_aClients[i].m_Score, gserver->m_apPlayers[i]->Warnings, pAuthStr);
+				str_format(aBuf, sizeof(aBuf), "id=%d addr=%s name='%s' score=%d aimMaxDist=%f experiment=%d warns=%d %s", i, aAddrStr,
```
Comment (black-roland) [inline src/engine/server/server.cpp:1466]: Maybe better to add separate `rcd_status` command for this? Like https://github.com/ddnet/ddnet/blob/fd9178262540c96b2a99a235dce77b5be864b300/src/engine/server/server.cpp#L2282

**Comment by black-roland** [comment]
How your experiment with `RajhCheatDetector::CheckSomethingElse()`? 🙂 I saw different values in status 😕

File: src/engine/server/server.cpp (hunk 1 above)
Comment (derselbst) [reply src/engine/server/server.cpp:1466]: In case we add another attribute, I agree. For now, I would leave it with that.

**Comment by derselbst** [comment]
changes available on bitbucket

**Pull request closed without merging**